docs/source/manifest.rst

```diff
@@ -0,0 +1,72 @@
+==============================================
+Understanding SMaHT Data Portal Manifest Files
+==============================================
+
+
+The preferred method to download data from the SMaHT data portal is to pass
+portal access credentials to the command provided in the ``smaht_manifest`` files.
+
+Most values in the manifest are single valued, but in some select scenarios multi-valued fields are separated by the pipe (``|``) character.
+
+Below are the columns listed in the ``manifest`` files as of the May 2024 data release.
+
+#. **File Download URL** - this URL, when navigated to while passing portal access credentials, will redirect to a pre-signed URL for downloading the associated file provided the user who made the request has access to the file.
+
+#. **File Accession** - this value is a server generated value specific to the SMaHT data portal, generally speaking it is unique with the exception that any associated extra files associated with the root file will have the same accession but different file extension. While you can reference files in data portal by this value, when communicating with others it is best to use the following field **File Name**.
+
+#. **File Name** - this value is the consortia-approved identifier for files, and is guaranteed to be a unique identifier encoding details about the file into the name. When downloading files, this will be the name of the file.
+
+#. **Size** - file size in bytes.
+
+#. **md5sum** - md5 of the file content.
+
+#. **Data Category** - this value tells you the file type, for example: Aligned Reads, Unaligned Reads or Variant Calls.
```

Comment:
I think Variant Calls is actually a Data Type, so I would replace it with SNV or Indel.

Reply:
Although this is called `Data Category`, it is actually accessing the data type field, so would `Variant Calls` be correct? and do you think should be renamed to `Data Type`?